### ISSUE TITLE
changeable template suffix (extension)

### ISSUE THREAD

**Issue by jusurb** [sebhoerl/Twig-netbeans#29]
Would be nice if you could change the default twig extension to something like html or phtml and get the syntax highlighting.
now if you don't use .twig template file extension you can't take advantage of this plugin

**Comment by Spone** [comment]
You can go to Tools > Options > Miscellaneous > Files, select the phtml extension and associate it to the Mime-Type text/twig.

**Comment by jusurb** [comment]
yeah, but when you create a new file ant select twig template, it uses the default extensions .twig instead of let say phtml
although the file says twig phtml template. I think it has something to do with changing the association

**Comment by Spone** [comment]
You're right, it does not enable new file creation with any other extension.
